package.json

```diff
@@ -38,7 +38,7 @@
     "istanbul": "~0.1.34",
     "mocha": "~1.13.0",
     "proxyquire": "0.5.1",
-    "should": "^4.0.4",
+    "should": "8.2.2",
```

Comment:
Should we re-create npm-shinkwrap.json due to the dependency has changed? (in a separted PR)

Reply:
Discussed with @AlvaroVega at skype. PR with  npm-shinkwrap.json modifications will come next.

Reply:
NTC